pkg/controllers/node/suite_test.go

```diff
@@ -66,7 +67,7 @@ var _ = Describe("Controller", func() {
 	})
 
 	AfterEach(func() {
-		node.Now = time.Now
```

Comment:
IMO time should work normally unless we explicitly need to control it.

Reply:
I doubt it's causing a problem right now, so not a blocking issue, but in general this will add variability - say the unit test process itself takes longer to schedule (on a heavily loaded machine running a lot of github actions for example), then some of these times might stretch longer. Probably not a big deal here since the code should generally do the right thing if things take "at least N seconds", but could lead to flakiness.